rs/p2p/quic_transport/src/lib.rs

```diff
@@ -225,7 +225,7 @@ impl Transport for QuicTransport {
             .ok_or(anyhow!("Currently not connected to this peer"))?
             .clone();
         peer.rpc(request).await.inspect_err(|err| {
-            info!(self.log, "{:#?}", err);
+            info!(every_n_seconds => 5, self.log, "{:#?}", err);
```

Comment:
Could we expand the log a bit as well? I remember seeing these logs and they're not very informative to me. Also do you think we need to pretty print the error? (I mean the `#`) 

```suggestion
            info!(every_n_seconds => 5, self.log, "Error sending rpc request to {}: {:?}", peer_id, err);
```

Reply:
Good points 👍